test/ui-test/lightspeedUiTestExplorer.ts

```diff
@@ -0,0 +1,271 @@
+import {
+  ActionsControl,
+  ActivityBar,
+  By,
+  ContextMenu,
+  EditorView,
+  InputBox,
+  ModalDialog,
+  SideBarView,
+  VSBrowser,
+  ViewControl,
+  ViewSection,
+  WebView,
+  WebviewView,
+  Workbench,
+} from "vscode-extension-tester";
+import {
+  getFixturePath,
+  expectNotification,
+  getModalDialogAndMessage,
+  getWebviewByLocator,
+  sleep,
+  updateSettings,
+  workbenchExecuteCommand,
+} from "./uiTestHelper";
+import { WizardGenerationActionType } from "../../src/definitions/lightspeed";
+import { PlaybookGenerationActionEvent } from "../../src/interfaces/lightspeed";
+import { expect } from "chai";
+import axios from "axios";
+
+export function lightspeedUIExplorerTest(): void {
+  describe("Test Lightspeed Explorer features", () => {
+    let workbench: Workbench;
+    let explorerView: WebviewView;
+    let modalDialog: ModalDialog;
+    let dialogMessage: string;
+    let sideBar: SideBarView;
+    let view: ViewControl;
+    let adtView: ViewSection;
+
+    before(async () => {
+      // Enable Lightspeed and open Ansible Light view on sidebar
+      workbench = new Workbench();
+      const settingsEditor = await workbench.openSettings();
+      await updateSettings(settingsEditor, "ansible.lightspeed.enabled", true);
+      await updateSettings(
```

Comment:
You can remove these `updateSettings()` calls and just add the following string at the top of the file: https://github.com/ansible/vscode-ansible/blob/main/test/ui-test/lightspeedUiTestPlaybookExpTest.ts#L1